csharp/Microsoft.Azure.Devices.Client.NetMF/Common/Security/SharedAccessSignatureBuilder.cs

```diff
@@ -0,0 +1,119 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.Azure.Devices.Client
+{
+    using Microsoft.Azure.Devices.Client.Extensions;
+    using System;
+    using System.Text;
+
+    sealed class SharedAccessSignatureBuilder
+    {
+        string key;
+
+        public SharedAccessSignatureBuilder()
+        {
+            this.TimeToLive = new TimeSpan(0, 20, 0);
```

Comment:
t's ok to make changes in the common code to avoid duplication by resorting to the most common denominator.